packages/types/src/lib/interfaces.ts

```diff
@@ -300,6 +309,31 @@ export interface JsonSignChainDataRequest {
   exp: number;
 }
 
+export interface JsonSignSessionKeyRequestV1 {
+  sessionKey: string;
+  authMethods: AuthMethod[];
+  pkpPublicKey?: string;
+  authSig?: AuthSig;
```

Comment:
You can't provide the AuthSig any more: https://github.com/LIT-Protocol/lit-assets/blob/aff82c4381139476aeaa6faa814069b184d38d39/rust/lit-node/src/endpoints/versions/v1.rs#L59